modules/money/daily.js

```diff
@@ -0,0 +1,76 @@
+/**
+ * @file daily command
+ * @author Sankarsan Kampa (a.k.a k3rn31p4nic)
+ * @license MIT
+ */
+const string = require('../../handlers/languageHandler');
+
+let recentUsers = [];
+const COOLDOWN = 24;
+
+exports.run = async (Bastion, message) => {
+  if (!recentUsers.includes(message.author.id)) {
+    let user = message.mentions.users.first() || message.author;
```

Comment:
Only the message author should be able to claim daily coins. One shouldn't be able to claim daily coins for others.

Reply:
I made it to work like tatsumaki